coding-standards/php.md

```diff
@@ -11,5 +11,24 @@ https://www.php-fig.org/psr/psr-12/
 [https://cheatsheetseries.owasp.org/cheatsheets/Laravel_Cheat_Sheet.html](https://cheatsheetseries.owasp.org/cheatsheets/PHP_Configuration_Cheat_Sheet.html)
 
 ## Enforcing tools and config
-- PHPCS - 
-- PHPSTAN -
+
+| Rule                                                                                                                          | Description                                                                                                                                      | URL                                                                                                                                                                                                                               |
+|-------------------------------------------------------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| SlevomatCodingStandard.TypeHints.UselessConstantTypeHint                                                                      | Checks for useless constant type hints in function and method signatures.                                                                           | [Link](https://github.com/slevomat/coding-standard/blob/master/doc/type-hints.md#slevomatcodingstandardtypehintsuselessconstanttypehint-)                                                                                         |
```

Comment:
You just add table.

But if you check https://github.com/OsmosysSoftware/dev-standards/blob/main/coding-standards/dotnet.md#editorconfig it gives the configuration file to enforce the rules.

Where is that?